imap_processing/glows/l1a/glows_l1a.py

```diff
@@ -353,31 +398,31 @@ def generate_histogram_dataset(
         time_data,
         name="epoch",
         dims=["epoch"],
-        attrs=ConstantCoordinates.EPOCH,
+        attrs=imap_object.get_variable_attributes("epoch"),
     )
     bin_count = 3600  # TODO: Is it always 3600 bins?
 
     bins = xr.DataArray(
         np.arange(bin_count),
         name="bins",
         dims=["bins"],
-        attrs=glows_cdf_attrs.bins_attrs.output(),
+        attrs=imap_object.get_variable_attributes("bins_attrs"),
     )
 
     hist = xr.DataArray(
         hist_data,
         name="histograms",
         dims=["epoch", "bins"],
         coords={"epoch": epoch_time, "bins": bins},
-        attrs=glows_cdf_attrs.histogram_attrs.output(),
+        attrs=imap_object.get_variable_attributes("histogram_attrs"),
     )
 
-    glows_attrs = glows_cdf_attrs.glows_l1a_hist_attrs.output()
-    glows_attrs["Data_version"] = data_version
+    glows_attrs_global = imap_object.get_global_attributes("imap_glows_l1a_hist")
+    glows_attrs_global["Data_version"] = data_version
```

Comment:
Hey, this line is a duplicate! I never noticed. Can you remove it, we're already setting the global attribute for Data_version elsewhere

Reply:
Actually you can get rid of `glows_attrs_global` entirely and just directly set `attrs = imap_object.get_global_attributes("imap_glows_l1a_hist")`